Remotion/Core/Collections.Caching.UnitTests/LazyLockingCachingAdapterTest.cs

```diff
@@ -181,14 +205,23 @@ public void TryGetValue_ValueFound ()
     [Test]
     public void TryGetValue_NoValueFound ()
     {
+      Lazy<Wrapper> outResult = null;
+
       _innerCacheMock
-          .Expect (mock => mock.TryGetValue (Arg.Is ("key"), out Arg<Lazy<Wrapper>>.Out (null).Dummy))
-          .Return (false)
-          .WhenCalled (mi => CheckInnerCacheIsProtected ());
+          .Setup (mock => mock.TryGetValue ("key", out outResult))
+          .Returns (false)
+          .Callback (
+              new OutDelegate (
+                  (string key, out Lazy<Wrapper> value) =>
+                  {
+                    value = null;
+                    CheckInnerCacheIsProtected();
+                  }))
```

Comment:
```suggestion
          .Callback (
              (OutDelegate) (string key, out Lazy<Wrapper> value) =>
              {
                value = null;
                CheckInnerCacheIsProtected();
              })
```
I haven't treied it, but this would normally be done as a cast instead of a ctor.